filament/src/details/Skybox.cpp

```diff
@@ -142,6 +142,8 @@ FMaterial const* FSkybox::createMaterial(FEngine& engine) {
                 assert_invariant(false);
 #endif
                 break;
+            case Engine::StereoscopicType::NONE:
```

Comment:
Can a normal code flow hit here? I think this shouldn't be hit because it'll include no skybox material otherwise. Can we add an assert here? But I feel like NONE doesn't make sense in this case.

Reply:
done